### PR TITLE
added missing filters

### PR DESCRIPTION
**Pull request by Darina007**

### CODE REVIEW

File: src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java [hunk 1]
```diff
@@ -163,42 +163,31 @@ public long createNotAssigned(long userId, Levels levelName) {
...
                 .status(Status.ASSIGNED)
                 .priority(priority)
                 .build();
-
         testsRepository.save(test);
-
         notificationService.create(NotificationType.TEST_ASSIGNED, test.getUser(), test);
-
         return test.getId();
     }
```
Comment (Anlord241) [inline src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java:185]: Please, don't remove ALL empty lines :)
They make code more readable

Comment (Darina007) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java:185]: but as for me it is very difficult to read the code as on lines 173-179 =)

Comment (maxim12321) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java:185]: Maybe let's edit IntelliJ IDEA code style properties?)
I propose these options (Editor -> Code Style -> Java -> Blank lines):

image

Comment (maxim12321) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java:185]: And yep, don't remove all empty lines, please))

Comment (AntonMaksimenkau) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java:185]: No, they don't. From time to time such code can be blocked by any style checkers.
Additionally, splitting a method by empty strings means that you're splitting the logic so you can split it into several methods. So, don't be shy to remove them.

**Review by AntonMaksimenkau — APPROVED** [review]

**Review by maxim12321 — APPROVED** [review]

**Pull request merged**